src/components/InternalAddress.vue

```diff
@@ -0,0 +1,160 @@
+<!--
+  - SPDX-FileCopyrightText: 2020 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<div>
+		<div v-for="domain in sortedDomains"
+			:key="domain.address">
+			{{ domain.address }}
+			{{ senderType(domain.type) }}
+			<ButtonVue type="tertiary"
+				class="button"
+				:aria-label="t('mail','Remove')"
```

Comment:
```suggestion
				:aria-label="t('mail', 'Remove')"
```

the usual styling nitpicks ;-)